pages/learn/manage/organizations.md

```diff
@@ -0,0 +1,104 @@
+
+<img alt="Invite Organization Member Modal" src="/img/common/app/invite_org_member.png">
+
+Depending on the role assigned, they will be able to access, see, and manage both apps and other members inside the organization. To change a member's role, navigate to the `Organization Members` tab in the sidebar. Next, change the `Organization role` of the desired user using the dropdown list against their username.
+
+<img alt="Change Organization Member Roles" src="/img/common/app/modify_org_roles.png" width="100%">
+
+
+## Creating teams in your organization
+
+Organization members can be grouped into teams. Teams provide a way to manage members according to their function. Only administrators can create and delete teams, and manage their members.
+
+To create a team, navigate to the `Organization Teams` page via the sidebar and click the `Create Team` button to enter a name for the team.
+
+<img alt="Create Team in Organization" src="/img/common/app/create_team.png" width="100%">
+
+Once the team is created, add members and grant the team access to applications by clicking on the `Manage members` and `Manage applications` buttons, respectively.
+
+<img alt="Organization Team Summary" src="/img/common/app/org_team_summary.png" width="100%">
+
+
+## Creating new applications under your organization
+
+With members invited to the organization, let's get started on creating new applications under an organization. Navigate to the `Organization Applications` tab in the sidebar to find the `Create application` button for creating a new application under your organization. Learn more about creating an application from the [getting started](getting-started) guide.
+
+<img alt="New Application under Organization" src="/img/common/app/create_app_under_org.png" width="100%">
+
+### Moving existing applications to an organization
+
+Moving an existing application to an organization is a common task accomplished by [transferring ownership](transferring-ownership) of your application.
+
+<img alt="Transfer Ownership" src="/img/common/app/transfer_ownership.png" width="100%">
+
+### Granting members access to applications
+
+Organization administrators can grant access to applications either directly for individual members or for teams. There are three application-level roles that can be assigned, [`Developer`](developer), [`Observer`](observer), and [`Operator`](operator), which grant varying levels of access. Administrators are implicitly granted full access to all applications of the organization.
+
+- **Granting access to applications for individual members** - Once you are done inviting members, click the username of the member you want to provide access to from the `Members` list. When the member's summary page opens, click `Manage applications` to see all applications available under the org. From there, click the `Add to applications` button to add the member to one or more applications available under the org with the desired role.
```

Comment:
Sorry, I think my previous suggestion is not proper English and maybe the following is better:

```suggestion
- **Granting individual members access to applications** - Once you are done inviting members, click the username of the member you want to provide access to from the `Members` list. When the member's summary page opens, click `Manage applications` to see all applications available under the org. From there, click the `Add to applications` button to add the member to one or more applications available under the org with the desired role.
```